core/metamodel/src/main/java/org/apache/isis/core/metamodel/adapter/ObjectAdapter.java

```diff
@@ -169,12 +162,6 @@
      */
     Oid getOid();
 
-//    /**
-//     * Since {@link Oid}s are now immutable, it is the reference from the
-//     * {@link ObjectAdapter} to its {@link Oid} that must now be updated.
-//     */
-//    void replaceOid(Oid persistedOid);
-
```

Comment:
could this be deleted rather than just commented out?

Reply:
yes